code/src/Components/OptionsWrapper.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import RadioButtonWrapper from './RadioButtonsWrapper'
+import DropDown from './DropDown'
+import DropDownWrapper from './DropDownWrapper'
+import TextInput from './TextInput'
+
+const OptionsWrapper = (props) => {
+    if (props.question.type === "radio") {
+        return (
+            <div className="options-wrapper">
+                <RadioButtonWrapper 
+                    question={props.question}
+                    toChange={props.toChange}
+                    source={props.source}
+                />
+            </div>
+        )
+    } else if (props.question.type === "dropDown") {
+        return (
+            <div className="options-wrapper">
+                <form className="stat-questions-wrapper">
+                    <DropDown 
+                        question={props.question}
+                        toChange={props.toChange}
+                        options={props.options}
+                        skillLevel={props.skillLevel}
+                        otherSkillLevel_a = {props.otherSkillLevel_a}
+                        otherSkillLevel_b = {props.otherSkillLevel_b}
+                        source={props.source}
+                    />
+                </form>
+            </div>
+        )
+    } else if (props.question.type === "text") {
+        return (
+            <div className="options-wrapper">
+                <form className="text-input-wrapper">
+                    <TextInput 
+                        
+                    />
+                </form>
+            </div>
+        )
+    } else {
+        return (
+            <div className="question">
+                What kinda question is this?
+            </div>
+        )
+    }
+}
+
+export default OptionsWrapper;
```

Comment:
Do I understand it correctly, that this is the only conditional rendering that is actually used at the moment. In App.js you have several but I do not see that they are actually being used.. Perhaps, I am just confused there.